Sources/BraintreeCore/BTAPIClientError.swift

```diff
@@ -36,6 +50,9 @@ public enum BTAPIClientError: Int, Error, CustomNSError, LocalizedError, Equatab
             
         case .failedBase64Encoding:
             return "Unable to base64 encode the authorization string."
+            
+        case .invalidAuthorization(let authorization):
+            return "Invalid authorization provided: \(authorization)."
```

Comment:
Is this same error message we surface on Android? Would be nice to align

Reply:
Updated to align a bit more. I do think it's helpful to pass through the auth the merchant passed in the message so we will diverge a bit there: https://github.com/braintree/braintree_ios/pull/1527/commits/984d65682ec27a56cc1882de4fb4e1f97ac3064f